scripts/Draconic-Evolution.zs

```diff
@@ -881,9 +875,9 @@ mods.avaritia.ExtremeCrafting.addShaped(<DraconicEvolution:sunDial>, [
 mods.avaritia.ExtremeCrafting.addShaped(<DraconicEvolution:grinder:3>, [
 [null, null, null, null, null, null, null, null, null],
 [null, null, null, null, null, null, null, null, null],
-[null, null, <ore:plateDraconium>, <ore:plateDraconium>, DraconicCore, <ore:plateDraconium>, <ore:plateDraconium>, null, null],
+[null, null, <ore:plateDraconium>, <ore:plateDraconium>, <ore:plateDraconium>, <ore:plateDraconium>, <ore:plateDraconium>, null, null],
 [null, null, <ore:plateDraconium>, <Thaumcraft:ItemSwordElemental>, <EnderIO:blockKillerJoe>, <Thaumcraft:ItemSwordElemental>, <ore:plateDraconium>, null, null],
-[null, null, <ore:plateDraconium>, <gregtech:gt.metaitem.01:32604>, <DraconicEvolution:generator:3>, <gregtech:gt.metaitem.01:32604>, <ore:plateDraconium>, null, null],
+[null, null, <ore:plateDraconium>, <gregtech:gt.metaitem.01:32604>, <EnderIO:blockCapBank:3>, <gregtech:gt.metaitem.01:32604>, <ore:plateDraconium>, null, null],
```

Comment:
if you stop using the Generator this machine/block is useless. MAybe we can change the Benerator recipe remove/replace the core instead

Reply:
I'm pretty sure it can't generate it's own power. It receives power from the outside. Or uses coal or whatever burnable to cook stuff normally, but slower than if it was using RF.

Reply:
ok